src/Monitors/Monitors.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Collection\Monitors;
+
+// commissioned datatype?
+
+readonly class Monitors
```

Comment:
Generally we make entity classes have singular names, since the job of this class is to represent a single record from the database